apps/web/modules/ui/components/input-combo-box/index.tsx

```
@@ -154,186 +150,225 @@
+                                    alt={opt.label}
+                                    width={24}
+                                    height={24}
+                                    className="shrink-0"
+                                  />
+                                )}
+                                <span className="flex-1 truncate text-slate-900">{opt.label}</span>
+                              </div>
+                            </DropdownMenuSubTrigger>
+                            <DropdownMenuSubContent alignOffset={4} className="w-48 p-1">
+                              {opt.children.map((child) => (
+                                <div className="flex flex-col">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)